R/options.R

```diff
@@ -62,9 +67,15 @@ polars_optreq$rpool_cap = list() # rust-side options already check args
 #' @param no_messages Hide messages.
 #' @param rpool_cap The maximum number of R sessions that can be used to process
 #' R code in the background. See Details.
+#' @param bigint_conversion How should Int64 values be handled when converting a
+#' polars object to R?
+#'
+#' * `"float"` (default) converts the values to Float64.
```

Comment:
I am confused. R doesn't have a float64 type, right?

I think it should be `numeric` instead of `float`.
For example, RPostgres' `bigint` option.
https://github.com/r-dbi/RPostgres/blob/f15f8f445def9e24ff98e4bfc910efcf705c85aa/R/dbConnect_RedshiftDriver.R#L5

Reply:
Actually "numeric" is misleading since an integer is also numeric in R. How about "double"?

``` r
is.numeric(1L)
#> [1] TRUE
is.double(1L)
#> [1] FALSE
```

Reply:
That sounds good, thanks.